package/DESCRIPTION

```diff
@@ -1,7 +1,6 @@
 Package: tlg.catalog.pkg
 Title: Tables, Listings and Graphs Catalog
 Version: 0.1.0.9009
-Date: 2024-01-26
```

Comment:
Just noticed you removed also Date... Was it also necessary?

Reply:
R CMD CHECK gave me note about it so I simply removed this :) It's not super critical